content/en/docs/reference/glossary/namespace.md

```diff
@@ -4,7 +4,7 @@ id: namespace
 date: 2018-04-12
 full_link: /docs/concepts/overview/working-with-objects/namespaces
 short_description: >
-  An abstraction used by Kubernetes to support multiple virtual clusters on the same physical cluster.
+  An abstraction used by Kubernetes to support isolation of groups of resources within a single {{< glossary_tooltip text="cluster" term_id="cluster" >}}.
```

Comment:
We don't use shortcodes inside glossary entry short descriptions. Can you rewrite this with just the plain word “cluster”?